src/Workspaces/CSharp/Portable/LanguageServices/CSharpSemanticFactsService.cs

```diff
@@ -300,7 +300,21 @@ public bool IsPartial(ITypeSymbol typeSymbol, CancellationToken cancellationToke
                 semanticModel.GetDeclaredSymbol(memberDeclaration, cancellationToken));
         }
 
-        public SymbolInfo GetSymbolInfo(SemanticModel semanticModel, SyntaxNode node, SyntaxToken token, CancellationToken cancellationToken)
+        public ImmutableArray<ISymbol> GetBestOrAllSymbols(SemanticModel semanticModel, SyntaxNode node, SyntaxToken token, CancellationToken cancellationToken)
+        {
+            switch (node)
+            {
+                case AssignmentExpressionSyntax assignment when token.Kind() == SyntaxKind.EqualsToken:
+                    return GetDeconstructionAssignmentMethods(semanticModel, node).As<ISymbol>();
```

Comment:
should we be doing something similar in VB?

Reply:
There is no deconstruction in VB yet